### ISSUE TITLE
Random nicknames messages when shift click items in inventory

### ISSUE THREAD

**Issue by UnitedCraft** [kiranhart/Auction-House#108]
**Describe the bug**
I found a strange problem and I wonder if it should be like this, but when we open /ah and shift-click on an item in the inventory, a message appears that the player only accepts the purchase of the entire... These messages are with random nicknames depending on the item type.
It depends on a item slot when You are shift clicking
I think the correct result here is that such an action should be blocked.

**To Reproduce**

- Describe how to reproduce this bug: EX.

1. Open /ah
2. shift-click on an item in the inventory

**Expected behavior**
No messages - blocked completly that action

**Comment by kiranhart** [comment]
What do you mean by nick names? I just receive the normal the player only accepts purchases purchases for the entire item message?

**Comment by UnitedCraft** [comment]
Yes, that's right, only the nickname with the message changes depending on the slot you perform the action on

> What do you mean by nick names? I just receive the normal the player only accepts purchases purchases for the entire item message?
